Assets/Scripts/Layers/Adapters/DataTypeAdapters/WFSGeoJSONImportAdapter.cs

```diff
@@ -130,12 +133,20 @@ private UriBuilder CreateLayerUri(string featureType, string sourceUrl)
             uriBuilder.SetQueryParameter("service", "WFS");
             uriBuilder.SetQueryParameter("request", "GetFeature");
             uriBuilder.SetQueryParameter("version", wfsVersion);
-            uriBuilder.SetQueryParameter("typeNames", featureType);
+            var typeNameFieldName = wfsVersion == "1.1.0" ? "typeName" : "typeNames";
```

Comment:
zou je een comment kunnen toevoegen waarom dit nodig is?

Reply:
Sterker nog, ik geloof dat dit elders in de code ook gedaan wordt; ik zal dat samenvoegen in een methode die dat beschrijft